src/amo/components/HomepageShelves/index.js

```diff
@@ -29,6 +29,8 @@ type InternalProps = {|
 export const HOMESHELVES_ENDPOINT_COLLECTIONS = 'collections';
 export const HOMESHELVES_ENDPOINT_SEARCH = 'search';
 export const HOMESHELVES_ENDPOINT_SEARCH_THEMES = 'search-themes';
+export const HOMESHELVES_ADDON_TYPE_EXTENSIONS = 1;
```

Comment:
As discussed, you don't need a new `const` for this. You can just use `ADDON_TYPE_EXTENSION`.